requirements/travis.txt

```diff
@@ -7,22 +7,22 @@
 appdirs==1.4.4            # via virtualenv
 certifi==2020.4.5.1       # via requests
 chardet==3.0.4            # via requests
-codecov==2.0.22           # via -r requirements/travis.in
+codecov==2.1.3            # via -r requirements/travis.in
 coverage==5.1             # via codecov
 distlib==0.3.0            # via virtualenv
 filelock==3.0.12          # via tox, virtualenv
 idna==2.9                 # via requests
 importlib-metadata==1.6.0  # via importlib-resources, pluggy, tox, virtualenv
 importlib-resources==1.5.0  # via virtualenv
-packaging==20.3           # via tox
+packaging==20.4           # via tox
 pluggy==0.13.1            # via tox
 py==1.8.1                 # via tox
 pyparsing==2.4.7          # via packaging
 requests==2.23.0          # via codecov
-six==1.14.0               # via packaging, tox, virtualenv
+six==1.15.0               # via packaging, tox, virtualenv
 toml==0.10.1              # via tox
-tox-battery==0.6.0        # via -r requirements/travis.in
```

Comment:
In this repo, `tox-battery` is already on latest version, do we need to constrain it as it is working fine on latest version in this repo?